### PR TITLE
✨ Burn structure + Tokens by status query

### PR DESCRIPTION
**Pull request by SurgeCode**
This PR includes 

-Restructuring of burn method into its own folder 
-Burn unit tests
-Burn documentation with frontend example
-Burn integration tests
-New query wrapper tokenByStatus
-tokenByStatus unit tests

tokenByStatus is used in the burn integration test so its also integration tested

### CODE REVIEW

File: packages/testing/tests/burn.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+test('burn token', async () => {
+  const accounts = ['mb_alice.testnet', 'mb_bob.testnet'];
+  const burnFromIndex = Math.random() > 0.5 ? 1 : 0;
+  const accountToBurnFrom = accounts[burnFromIndex];
+  const keyStore = await authenticatedKeyStore([accountToBurnFrom]);
+  const signingAccount = await connect(accountToBurnFrom, keyStore);
+  const token = await ownedTokens(accountToBurnFrom, { limit: 1 });
+  
+  if (!token) {
+    throw `${accountToBurnFrom} ran out of tokens to transfer! Mint some more...`;
```
Comment (cif) [inline packages/testing/tests/burn.integration.test.ts:19]: Update console

Comment (SurgeCode) [reply packages/testing/tests/burn.integration.test.ts]: fixed

File: packages/sdk/src/burn/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+[//]: # `{ "title": "burn", "order": 0.1 }`
+
+# Burn Tokens
+
+Burn one or more tokens for a given smart contract id.
+
+**As with all new SDK api methods, this call should be wrapped in [execute](../#execute) and passed a signing method
```
Comment (cif) [inline packages/sdk/src/burn/README.md:7]: owned tokens also has this formatting issue, missing ** on the end

**Comment by SurgeCode** [comment]
> Looks good! After standup tomorrow, remind me to demo how the docs are working for everyone to spot check that from PRs here, I tweaked the ordering a bit to make burn docs show up

Sounds good, I saw what you did and it makes sense

**Pull request merged**